openapi/api.yml

```diff
@@ -155,6 +157,67 @@ paths:
                 status: 'error'
                 message: 'Could not save Score for application sub-criteria'
 
+  '/flag':
+    get:
+      tags:
+        - flags
+      summary: get flags for an application
+      description: get flags for an application
+      operationId: api.routes.get_flags_for_application
+      responses:
+        200:
+          description: SUCCESS - Flags for an Application
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: 'components.yml#/components/schemas/Flag'
+      parameters:
+      - name: application_id
+        in: query
+        schema:
+          type: string
+        required: true
+    post:
+      tags:
+        - flags
+      summary: post a flag for an application
+      description: post a flag for an application
+      operationId: api.routes.post_flag_for_application
+      requestBody:
+        description: Flag creation parameters
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: 'components.yml#/components/schemas/PostFlag'
+            example:
+              justification: 'justification for flagging'
+              section_to_flag: 'information on the section to flag'
+              flag_type: 'FLAGGED'
+              application_id: 'r3p1ac33-7412-4991-c4t1-c41cbf4d3bbd'
+              user_id: 'r3p1ac33-7412-4991-c4t1-c41cbf4d3bbd'
+      responses:
+        201:
+          description: SUCCESS - Flag for Application Submitted
+          content:
+            application/json:
+              schema:
+                type: object
+                items:
+                  $ref: 'components.yml#/components/schemas/Flag'
+        401:
```

Comment:
Is this Supposed to ve "Could not Save Flag"?

Reply:
Good eye, have fixed in https://github.com/communitiesuk/funding-service-design-assessment-store/pull/75/commits/b6b34c0ed88dcf65c191572addf4a181fa67a8c3